frontend/src/App.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from 'react';
+import './App.css';
+import {
+  BrowserRouter, Routes, Route, Navigate,
+} from 'react-router-dom';
+import Header from './components/Header';
+import PhysiciansList from './components/Physicians/PhysiciansList';
+import Physician from './components/Physicians/Physician';
+import AddPhysicianForm from './components/Physicians/AddPhysicianForm';
+import DeletePhysician from './components/Physicians/DeletePhysician';
+import BookAppointmentForm from './components/Appointments/BookAppointmentForm';
+import AppointmentsList from './components/Appointments/AppointmentsList';
+import Login from './components/login/Login';
+import Register from './components/register/Register';
+import './components/Navbar.css';
+import useAuth from './state';
+
+function App() {
```

Comment:
- [ ] For many components and functions, you are using the keyword `function`.  Kindly refactor those functions and components to ES6 arrow functions.

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions.

You can make a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry.

That way, your app will comply with the latest standards 💪